app/locales/de/messages.po

```diff
@@ -498,11 +502,16 @@ msgstr "Interaktive Filter hinzufügen"
 msgid "controls.section.interactiveFilters.dataFilters"
 msgstr "Datenfilter"
 
+#: app/configurator/map/map-chart-options.tsx:154
+#: app/configurator/map/map-chart-options.tsx:341
+msgid "controls.section.map.noGeoDimensions"
+msgstr "In diesem Datenset können keine geografischen Dimensionen angezeigt werden!"
```

Comment:
Maybe we should not have a "!" at the end, I found it a bit "violent".

Reply:
done in another PR 👍